regression/goto-harness-gb-input/static-symbol-referencing/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+aws_byte_buf_append_harness.goto
+--harness-type call-function --harness-function-name generated_harness_test_function --function aws_byte_buf_append
+^EXIT=6$
+^SIGNAL=0$
+--
+^CONVERSION ERROR$
+--
+For this particular error, we care mostly that goto-harness
```

Comment:
:pick: Probably worth adding a positive regex (e.g. verification successful) just in case the error message for conversion error ever changes

Reply:
I'm not sure I agree with this tbh.

I see your point, but the whole essence of this test is to test the parser - or more accurately, that the parsing of this file is successful and the file doesn't contain any erroneous input.

If I add a positive regex (i.e. `VERIFICATION SUCCESSFUL`), then I'm also depended on the analysis part to have a specific output, something that I feel is extraneous for this specific issue.

I'll have a think about improving the test though.

Reply:
Well maybe not verification successful, but some check that the parsing has been successful. Much better to have a test that needs periodic updating when it fails, than a test that starts spuriously passing

Reply:
I think I found a way: I am including some of the pipeline status messages in the `test.desc`, like `converting`, `type-checking`, etc..

This way, it will fail both on the assertion failure, and if it didn't go through all the appropriate stages.

Reply:
Agree with @thk123 

It may be testing for the parsing but `VERIFICATION SUCCESSFUL` turns up in so many of our other tests that it is hardly an unreasonable extra condition.

Reply:
It is better to test more than you need and to have a more fragile test than to have something that may just start silently passing even when wrong.